css/app/datepicker.css

```diff
@@ -89,27 +89,35 @@
 	border-left: 0;
 }
 
-#datepicker table tbody tr.uib-weeks td:last-child button {
-	border-right: none;
+.today-hilight {
+	border-radius: 50%;
+	background-color: #e6e6e6;
 }
 
-#datepicker table tbody tr.uib-weeks:last-child td button {
-	border-bottom: none;
+#datepicker table tbody tr td button {
+	border: none;
 }
 
 #datepicker table tbody button {
 	max-width: 34px; /* Hack for Firefox for overflowing tables */
 }
 
 #datepicker table tbody button.active {
-	background-color: rgba(240, 240, 240, .9);
+	border-radius: 50%;
+	background-color: #0082c9;
```

Comment:
> they should :) , but it should be in another issue.

Yes :)

I would vote to merge this now and fix the scss / primary color issue in a follow up PR :)